services/QuillLMS/client/app/bundles/Evidence/components/studentView/container.tsx

```diff
@@ -475,11 +476,19 @@ export const StudentViewContainer = ({ dispatch, session, isTurk, location, acti
       const highlightIncludesElement = strippedPassageHighlights.find(ph => ph.includes(stringifiedInnerElements)) // handles case where passage highlight spans more than one paragraph
       const elementIncludesHighlight = strippedPassageHighlights.find(ph => stringifiedInnerElements.includes(ph)) // handles case where passage highlight is only part of paragraph
       if (highlightIncludesElement) {
-        return <p><span className="passage-highlight">{stringifiedInnerElements}</span></p>
+        return (
+          <p>
+            <span className="passage-highlight">
+              <span className="sr-only">(yellow underlined text begins here)</span>
+              {stringifiedInnerElements}
+              <span className="sr-only">(yellow underlined text ends here)</span>
+            </span>
+          </p>
+        )
       }
       if (elementIncludesHighlight) {
         let newStringifiedInnerElements = stringifiedInnerElements
-        strippedPassageHighlights.forEach(ph => newStringifiedInnerElements = newStringifiedInnerElements.replace(ph, `<span class="passage-highlight">${ph}</span>`))
+        strippedPassageHighlights.forEach(ph => newStringifiedInnerElements = newStringifiedInnerElements.replace(ph, `<span class="passage-highlight"><span class="sr-only">(yellow underlined text begins here)</span>${ph}<span class="sr-only">(yellow underlined text ends here)</span></span>`))
```

Comment:
Yeah, I thought about just making it a shared component here, but it's a little weird because in the first instance we're rendering it as a JSX element and here it's a string (hence the `className` vs `class` difference) that then gets converted into HTML, so there actually wasn't a super easy way to do that.

Reply:
Yeah, that makes sense.  Giving ourselves time to see how we want to use something like this before trying to make shared component makes sense.